vignettes/analysis_examples.Rmd

```diff
@@ -0,0 +1,173 @@
+---
+title: "Example Analysis Workflow"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+library(DIMPLE)
+library(tidyverse)
+```
+
+## Setting up the data
+
+In this vignette, we will show an example of an analysis workflow that could be performed using the `DIMPLE` package.
+We will be using a simulated point pattern dataset, along with simulated patient metadata that has a continuous `age` covariate and categorical `group` covariate. We will demonstrate how to analyze this data using the JSD distance metric. The below steps for seting up the `MltplxExperiment` object are similar to those performed in other vignettes, so we won't explain this process again here. 
+
+```{r}
+# Create an experiment with intensities, distance matrices and patient metadata
+set.seed(1234)
+N <- 30000
+n_slides <- 30
+cell_x_values = runif(N, 0, 600)
+cell_y_values = runif(N, 0, 600)
+cell_marks = sample(c("Tumor", "Immune", "Stroma","Other"), N, replace = TRUE)
+slide_ids = rep(paste("Slide", 1:n_slides), each = 1000)
+
+slides <- unique(slide_ids)
+n_patients <- 10
+n_groups <- 2
+patient_ids <- rep_len(paste0("P",1:n_patients),n_slides)
+
+metadata <- tibble::tibble(slide_id=slides,
+                   patient_id=patient_ids)
+
+groups <- sample(rep_len(sample(paste0("G",1:n_groups)),n_patients))
+
+ages <- runif(n_patients,min=45,max=100)
+
+gp_tb <- tibble::tibble(patient_id=unique(patient_ids),group=groups,age=ages)
+metadata <- dplyr::left_join(metadata,gp_tb,by = "patient_id")
+print(metadata)
+
+mltplx_experiment = new_MltplxExperiment(x = cell_x_values,
+                                  y = cell_y_values,
+                                  marks = factor(cell_marks),
+                                  slide_id = slide_ids,
+                                  ps = 30, bw = 40,
+                                  dist_metric = jsd,
+                                  metadata = metadata)
+
+```
+
+Next we can add information on how the JSD between cell types changes in different areas of higher and lower tumor density, by using the `update_qdist` function. This function takes in the `MltplxExperiment` object, the distance metric of choice, a cell type by which the image is partitioned on (by the density of that cell type), as well as the quantiles that the mask density should be partitioned into.
```

Comment:
It might be worth adding another sentence or two to clarify how exactly that partitioning happens

Reply:
And just generally clarifying what this type of quantile distance analysis is